src/de/darmstadt/tu/crossing/CipherInputStream.cryptsl

```diff
@@ -0,0 +1,30 @@
+SPEC javax.crypto.CipherInputStream
+OBJECTS
+    byte[] b;
+    int off;
+    int len;
+    long n;
+    java.io.InputStream is;
+    javax.crypto.Cipher c;
+
+EVENTS
+    c1: CipherInputStream(is, c);
+
+    r1: read();
+    r2: read(b);
+    r3: read(b, off, len);
+    Reads := r1 | r2 | r3;
+
+    c: close();
+
+FORBIDDEN
+    CipherInputStream(is) => c1;
```

Comment:
This constructor is protected. We don't need to prohibit its use. Same for CipherOutputStream.

Reply:
Because there were two constructors that we could use for each CipherInputStream and CipherOutputStream, I thought to only permit using the constructor that takes as input the InputStream and Cipher objects, since would be more secure. Anyway, I will delete the FORBIDDEN part for both Cipher I/O Stream